### PR TITLE
docs(react-swatch-picker): fixed icon in HC and image example

### PR DESCRIPTION
**Pull request by ValentinaKozlova**
## Previous Behavior
Icon in HC
image

Image example
image

## New Behavior
Icon
image
Image:
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/Avatar-mount_base.html">629</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/Avatar-mount.html">654</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/Button-mount_base.html">313</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/Button-mount.html">294</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/Field-mount_base.html">1131</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/Field-mount.html">1123</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/FluentProvider-mount_base.html">697</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/FluentProvider-mount.html">727</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/FluentProviderWithTheme-mount_base.html">78</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/FluentProviderWithTheme-mount.html">80</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">33</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">40</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">73</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">74</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/MakeStyles-mount_base.html">869</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/MakeStyles-mount.html">864</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/Persona-mount_base.html">1775</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/Persona-mount.html">1704</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/SpinButton-mount_base.html">1440</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/SpinButton-mount.html">1411</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/SwatchPicker-mount_base.html">1523</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31547/perf-test-react-components/SwatchPicker-mount.html">1567</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-swatch-picker/stories/SwatchPicker/ColorSwatchVariants.stories.tsx [hunk 1]
```diff
@@ -8,6 +8,11 @@ const useStyles = makeStyles({
     display: 'flex',
```
Comment (fabricteam) [inline packages/react-components/react-swatch-picker/stories/SwatchPicker/ColorSwatchVariants.stories.tsx:8]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31547&commitId=d44a78803f5de1980ede8f672459d683d65b8773&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - High Contrast.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Review by mainframev — APPROVED** [review]
LGTM 👍🏻

**Review by tudorpopams — APPROVED** [review]

**Pull request merged**